apps/client/src/pages/DevHomePage.js

```diff
@@ -1,17 +1,44 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import { Link } from "react-router-dom";
 
 import Layout from "../components/Layouts/DefaultLayout";
-import { isProduction, topics } from "../config";
+import { Flow, isProduction, topics } from "../config";
 import { geturl, routes } from "../routes";
 
 const DevHomePage = () => {
+  const [config, setConfig] = useState([]);
+
   if (isProduction) {
     localStorage.setItem("doNotTrack", "true");
   }
+  useEffect(() => {
+    fetch(`${window.location.origin}/sttr/topics.json`)
+      .then((res) => res.json())
+      .then(setConfig);
```

Comment:
You are using `setConfig` without passing a value? Does that work?

Reply:
Yeah. The json is parsed from `res`. The output is then sent to the next `then` so basically the same as `.then((res) => setConfig(await res.json()))` but more functional this way.
Not it doesn't call setConfig with `()` it just passes the function to the next `then`.